src/plugins/intel_cpu/src/nodes/executors/acl/acl_conv.cpp

```diff
@@ -0,0 +1,149 @@
+    : weightScale(attrs.dqScales) {
+    MemoryDescPtr srcMemPtr = memory.at(ARG_SRC_0)->getDescPtr();
+    MemoryDescPtr weiMemPtr = memory.at(ARG_WEI)->getDescPtr();
+    MemoryDescPtr dstMemPtr = memory.at(ARG_DST)->getDescPtr();
+
+    Shape weiShape = weiMemPtr->getShape();
+    Shape srcShape = srcMemPtr->getShape();
+    Shape dstShape = dstMemPtr->getShape();
+
+    const auto with_groups = static_cast<const int>(weiShape.getRank() == srcShape.getRank() + 1);
+    const int kh = weiShape.getDims()[with_groups + srcShape.getRank() - 2];
+    const int kw = weiShape.getDims()[with_groups + srcShape.getRank() - 1];
+    const int oc = dstShape.getDims()[1];
+
+    weightsInfo = arm_compute::WeightsInfo(false, kw, kh, oc, false, arm_compute::WeightFormat::UNSPECIFIED);
+    auto paddingLeft = (attrs.paddingL.size() >= 2U) ? attrs.paddingL[1] : attrs.paddingL[0];
+    auto paddingRight = (attrs.paddingR.size() >= 2U) ? attrs.paddingR[1] : attrs.paddingR[0];
+    auto paddingTop = (attrs.paddingL.size() >= 2U) ? attrs.paddingL[0] : 0;
+    auto paddingBottom = (attrs.paddingR.size() >= 2U) ? attrs.paddingR[0] : 0;
+    padStrideInfo = arm_compute::PadStrideInfo(attrs.stride[0],
+                                               attrs.stride[1],
+                                               paddingLeft,
+                                               paddingRight,
+                                               paddingTop,
+                                               paddingBottom,
+                                               arm_compute::DimensionRoundingType::FLOOR);
+    dilation = arm_compute::Size2D(attrs.dilation[1] + 1, attrs.dilation[0] + 1);
+
+    if (attrs.postOps.size() == 1) {
+        if (const auto* const activation = std::any_cast<ActivationPostOp>(attrs.postOps.data())) {
+            activationLayerInfo = getActivationLayerInfo(convertToEltwiseAlgorithm(activation->type()),
+                                                         activation->alpha(),
+                                                         activation->beta(),
+                                                         activation->gamma());
+        } else if (const auto* const fq = std::any_cast<FakeQuantizePostOp>(attrs.postOps.data())) {
+            fqInputScale = fq->inputScale();
+            fqInputShift = fq->inputShift();
+            fqOutputScale = fq->outputScale();
+            fqOutputShift = fq->outputShift();
+            if (fqOutputScale.size() == 1 && fqOutputScale[0] == 1.0F && fqOutputShift.size() == 1 &&
+                fqOutputShift[0] == std::trunc(fqOutputShift[0])) {
+                for (auto& v : fqInputShift) {
+                    v += fqOutputShift[0];
+                }
+                fqOutputShift.clear();
+            }
+        }
+    } else {
+        DEBUG_LOG("ACLConvolutionExecutor: post op is not applied!");
+    }
```

Comment:
I still think that we should throw an exception in this case. Because if the post ops were fused into node, but were not applied, than we will most likely loose accuracy.
It is much easier to debug functional issue than accuracy one

This comment is actual for
1. `postOps` size
2. `attrs.postOps.data()` is not `FakeQuantizePostOp` or `ActivationPostOp`